praw/models/reddit/comment.py

```diff
@@ -50,15 +60,18 @@ def submission(self, submission):
         for reply in getattr(self, 'replies', []):
             reply.submission = submission
 
-    def __init__(self, reddit, id=None,  # pylint: disable=redefined-builtin
-                 _data=None):
+    def __init__(self, reddit, id=None, url=None, _data=None):
+        # pylint: disable=redefined-builtin
```

Comment:
Can you move the `url=None` down to the second line so that the pylint comment still works?